extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/README.md

```diff
@@ -30,3 +32,15 @@ The following configuration of the service is provided.  Additional configuratio
 |---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
 | kafka.bootstrap.servers                     | Specifies the kafka bootstrap server when using kafka for messaging                                                                                           | Any valid URI                                                                 |
 | mp.messaging.incoming.pipeline-invocation.* | Specifies and configures the smallrye connector to use.  Supported connectors are `smallrye-amqp`, `smallrye-kafka`, `smallrye-mqtt`, and `smallrye-rabbitmq` | See xref:messaging-details.adoc[the Messaging documentation] for more details |
+
+
+## Migrated Properties
+The service account for pipeline invocation service is separated from spark operator. It's now configured solely for the pipeline invocation chart.
```

Comment:
A: I don't think we need to state this.  Anyone outside of baseline developers won't have really even known it used to be this way, and I don't think people would expect that it would be this way.  Probably good enough just to mention this change in the release notes and not here.